src/pages/Employees.jsx

```diff
@@ -46,22 +47,41 @@ const Employees = () => {
       field: "lastLogin",
       headerName: "Last Login",
       flex: 1,
-      filter: true
+      filter: true,
     },
     {
       field: "actions",
       headerName: "Actions",
       width: "310px",
-      cellRenderer: EmployeeButtonsRenderer
-    }
+      cellRenderer: EmployeeButtonsRenderer,
+      cellRendererParams: { setAction, setOpen, setEmployee },
+    },
   ];
   return (
     <div>
       <h2>Employee Management</h2>
-      <Button variant="outlined" sx={{ mb: 1 }} onClick={() => setOpen(true)}>
+      <Button
+        variant="outlined"
+        sx={{ mb: 1 }}
+        onClick={() => {
+          setAction("Add");
+          setOpen(true);
+        }}
+      >
         Add Employee
       </Button>
-      <AddEmployee open={open} onClose={() => setOpen(false)} />
+      {open && (
+        <AddEmployee
+          open={open}
+          onClose={() => {
+            setOpen(false);
+            setAction("");
+            setEmployee(null);
```

Comment:
Delvin wants properties happening together to be grouped into a single named object, eg
setObject({ open:false,action=""employee:null})